django/contrib/postgres/aggregates/statistics.py

```diff
@@ -0,0 +1,80 @@
+from django.db.models import FloatField, IntegerField
+from django.db.models.aggregates import Aggregate
+
+__all__ = [
+    'CovarPop', 'Corr', 'RegrAvgX', 'RegrAvgY', 'RegrCount', 'RegrIntercept',
+    'RegrR2', 'RegrSlope', 'RegrSXX', 'RegrSXY', 'RegrSYY', 'StatAggregate',
+]
+
+
+class StatAggregate(Aggregate):
+    def __init__(self, y, x, output_field=FloatField()):
```

Comment:
Why y, x instead of x, y? Is this from PostgreSQL's function definition. If so, OK, if not, then I think the order should be x, y.